frontend/src/components/eMIB/Email.jsx

```diff
@@ -55,10 +63,10 @@ class Email extends Component {
     return (
       <div style={styles.email}>
         <div style={styles.header}>
-          <h6 style={styles.emailId}>
+          <h2 style={styles.emailId} className="emailId">
```

Comment:
Why do you have **'emailId'** in both attributes, _style_ **and** _className_? Is that referring to the same thing?

Reply:
Good catch... I started with it defined in the css, then moved it to the style instead. so this is longer needed. Fixed in 5236a00